.github/workflows/docs.yml

```diff
@@ -37,3 +37,14 @@ jobs:
           branch: gh-pages # The branch the action should deploy to.
           folder: docs/out # The folder the action should deploy.
           clean: true # Automatically remove deleted files from the deploy branch
+
+      - name: Algolia
+        uses: wangchucheng/algolia-uploader@master
+        with:
+          app_id: ${{secrets.ALGOLIA_APPLICATION_ID}}# You can store token in your project's 'Setting > Secrets' and reference the name here. Such as ${{ secrets.ALGOLIA_ADMIN_KEY }}
+          admin_key: ${{secrets.ALGOLIA_SEARCH_API_KEY}}
+          # The index name. 
+          index_name: ${{secrets.ALGOLIA_INDEX_NAME}}
+          # The index file path relative to repo root.
+          index_file_path: ${{secrets.ALGOLIA_INDEX_FILE_PATH}}
```

Comment:
There is no secret `ALGOLIA_INDEX_FILE_PATH `. You should put the path here directly.

Reply:
resolve